script.js

```diff
@@ -95,16 +95,33 @@ jQuery(function() {
     if (jQuery('#configUserChoice').attr('content') == '0'){
         localStorage.setItem('configUserChoice', '0');
     }
+	
+	if (jQuery('#configAutoDark').attr('content') == '1'){
+        localStorage.setItem('configAutoDark', '1');
+		localStorage.setItem('changedAutoDark', '0');
```

Comment:
Unfortunately, it does not work for me. changedAutoDark is set back to 0 every time a new page is loaded.

Reply:
I think it is now resolved.